src/flask_login/utils.py

```diff
@@ -179,11 +179,11 @@ def login_user(user, remember=False, duration=None, force=False, fresh=True):
     user_id = getattr(user, current_app.login_manager.id_attribute)()
     session["_user_id"] = user_id
     session["_fresh"] = fresh
+    session["_remembered_login"] = False
     session["_id"] = current_app.login_manager._session_identifier_generator()
 
     if remember:
         session["_remember"] = "set"
-        session["_remembered_login"] = True
```

Comment:
It's a good idea to reset the variable for regular logins. 👍🏻 However, in order to keep this new utility function working as expected, I would suggest to restore this line. Otherwise it won't know whether a login is remembered right after logging in.